cuda/private/repositories.bzl

```diff
@@ -41,7 +41,47 @@ def detect_cuda_toolkit(repository_ctx):
         A struct contains the information of CUDA Toolkit.
     """
     cuda_path = repository_ctx.attr.toolkit_path
-    if cuda_path == "":
+    if cuda_path == "" and repository_ctx.attr.cuda_version != "":
```

Comment:
Why duplicate all these line of code?

Reply:
if not set cuda_path and set cuda version ,  mean CUDA_PATH  have precedence over CUDA_VERSION